Assets/Scripts/PhysicsPlayerCharacter.cs

```diff
@@ -0,0 +1,178 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class PhysicsPlayerCharacter : MonoBehaviour {
+
+    [SerializeField]
```

Comment:
For your Serialized Fields, I'd put a #region at the top, and #endregion at the bottom. It makes it collapsable for organization purposes.

Reply:
done